File: cfgs/java_cfgs_foo/COUNT_NUMBER_PAIRS_N_B_N_GCD_B_B.java
Nodes
1;special;;;
2;statement;int k = foo_1;14;
3;statement;int imin = 1;15;
4;statement;int ans = 0;16;
5;conditional;foo_1 <= foo_2;17;
6;statement;int imax = foo_1 / foo_2;18;
7;statement;ans += foo_1 * ((foo_2 - foo_3) + 1);19;
8;statement;imin = foo_1 + 1;20;
9;statement;k = foo_1 / foo_2;21;
10;statement;return foo_1;23;
11;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;
4;10;6;
5;7;
6;8;
7;9;
8;5;
9;11;
10;

File: cfgs/pythoncfgs/COUNT_NUMBER_PAIRS_N_B_N_GCD_B_B.py
Nodes
1;statement;foo_1 = foo_2;8;
2;statement;foo_1 = 1;9;
3;statement;foo_1 = 0;10;
4;conditional;foo_1 <= foo_2;11;
5;statement;return foo_1;16;
6;statement;foo_1 = foo_2 / foo_3;12;
7;statement;foo_1 += foo_2 * (foo_3 - foo_4 + 1);13;
8;statement;foo_1 = foo_2 + 1;14;
9;statement;foo_1 = foo_2 / foo_3;15;
10;exit;;
Edges
0;2;
1;3;
2;4;
3;5;6;
4;10;
5;7;
6;8;
7;9;
8;4;
9;